docs/cloud/rahti/agreements/sla.md

```diff
@@ -1,8 +1,8 @@
-# Rahti Container Cloud Service – Service Level Agreement
+# Rahti Container Cloud Service Service Level Agreement
```

Comment:
Were the SLAs and ToUs taken directly from here? The master source is this, and it should be kept up to date, otherwise changes will be overwritten on the next upgrade
https://github.com/CSCfi/rahti-user-guide/tree/master/docs/agreements

Reply:
I am not sure, @shukapoo did this one.

Reply:
Shubham resolved with: [50262c2](https://github.com/CSCfi/csc-user-guide/pull/601/commits/50262c296947f1d7d70a2348722ba17e03c1e076)